### PR TITLE
[Form] Fixed the layout in the forms page

### PR DESCRIPTION
**Pull request by TimoBakx**
By moving the configuration tabs outside the versionadded block.

### CODE REVIEW

File: forms.rst [hunk 1]
```diff
@@ -560,46 +560,51 @@ To see the second approach - adding constraints to the form - and to
 learn more about the validation constraints, please refer to the
 :doc:`Symfony validation documentation </validation>`.
 
+Friendlier validation messages
```
Comment (wouterj) [inline forms.rst]: In the docs we use a "title case" with all words capitalized except from closed-class words.

I would also propose a more neutral title like "Form Validation Messages"

File: forms.rst [hunk 2]
```diff
@@ -560,46 +560,51 @@ To see the second approach - adding constraints to the form - and to
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
 .. versionadded:: 5.2
 
-    In Symfony 5.2, the form validation messages have been rewritten to be more
-    user-friendly. Set the ``legacy_error_messages`` option to ``false`` to
-    enable these new messages:
+    The ``framework.form.legacy_error_messages`` option was added in Symfony 5.2
```
Comment (wouterj) [inline forms.rst]: We always try to use "[...] was introduced in Symfony 5.2." in all our versionadded directives.

File: forms.rst [hunk 3]
```diff
@@ -560,46 +560,51 @@ To see the second approach - adding constraints to the form - and to
...
-                'form' => [
-                    'legacy-error-messages' => false,
-                ],
-            ]);
+    .. code-block:: php
+
+        // config/packages/framework.php
+        $container->loadFromExtension('framework', [
+            'form' => [
+                'legacy-error-messages' => false,
```
Comment (wouterj) [inline forms.rst]:
```suggestion
                'legacy_error_messages' => false,
```

**Comment by TimoBakx** [comment]
@wouterj, thanks for the feedback. I included all your suggestions.

**Comment by wouterj** [comment]
Thank you! After merging, I reworded the paragraph a little bit in c4b9fc4 Please let me know if you disagree with the changes :)

**Comment by TimoBakx** [comment]
Sounds good to me!

**Pull request merged**